assemble/bin/accumulo-cluster

```diff
@@ -115,15 +119,19 @@ function parse_config {
     echo "INFO: ${NUM_TSERVERS} tservers will be started per host"
   fi
 
-  # shellcheck disable=SC2153
-  if [[ -z $NUM_SSERVERS ]]; then
-    echo "INFO: ${NUM_SSERVERS} sservers will be started per host"
-  fi
-
-  if [[ -z $NUM_COMPACTORS ]]; then
-    echo "INFO: ${NUM_COMPACTORS} compactors will be started per host"
-  fi
+  for group in $SSERVER_GROUPS; do
```

Comment:
You might still need the shellcheck comment. It suppresses  https://www.shellcheck.net/wiki/SC2153

Reply:
After removing it I didn't see shellcheck throwing errors about it. 
I'm curious if it was previous flagging on `SSERVER_GROUPS` vs `TSERVER_GROUPS` but maybe now there's enough other references that it doesn't trigger.